### DISCUSSION TITLE
I want to modify the Docker version

### DISCUSSION THREAD

**Discussion by dev4ma** [postalserver/postal#1988] [Help with using Postal]
Hello everyone, 
I want to know how to modify the Docker version
What I need to modify is the header: **( Received: from )**

I want to change the domain and IP indicated in the following screenshot in green

![Screenshot 2022-05-12 004048](https://user-images.githubusercontent.com/56938522/167959338-83a12812-7ffc-43ed-9d74-55e07a39ba9f.png)

**Comment by dev4ma** [comment]
up

**Comment by willpower232** [comment]
You can either modify the docker-compose.yml file to override whichever file is adding the line you don't want or make your own docker image.

**Comment by dev4ma** [comment]
Please, I need more clarification on how to do this
thank you for helping me

**Comment by dev4ma** [comment]
up
